### PR TITLE
Add force_ctas_schema to query model when enabled

### PR DESCRIPTION
**Pull request by vera-liu**
Before:
 - force_ctas_schema is not passed to Query model when enabled, the output column of Query Table  does not receive schema from backend unless it is set in query editor

After:
 - when force_ctas_schema is set, it gets stored in query, and will be displayed in output column of QueryHistory if the query was run using ctas

Tested in staging: 
 - Use case one: run query with selected schema in left pane
1. set schema in sql editor left pane
2. run a query
3. schema is shown in output column of Query History

 - Use case two: run query with ctas on a database with force_ctas_schema
 1. set schema in database view
 2. run a query using ctas
 3. schema.table_name is shown in output column of Query History

 - Use case three: run query with selected schema and ctas 
1. set schema in sql editor left pane
2. run a query using ctas
3. schema.table_name is shown in output column of Query History

Todo:
 - [https://github.com/airbnb/superset/issues/1850](url)
needs-review @bkyryliuk @mistercrunch

### CODE REVIEW

File: superset/views.py [hunk 1]
```diff
@@ -2383,12 +2383,16 @@ def table_accessible(database, full_table_name, schema_name=None):
                 get_datasource_access_error_msg('{}'.format(rejected_tables)))
         session.commit()
 
+        select_as_cta = request.form.get('select_as_cta') == 'true'
+        if select_as_cta and mydb.force_ctas_schema:
+            schema = mydb.force_ctas_schema
```
Comment (bkyryliuk) [inline superset/views.py]: I am afraid it will break sql lab.
schema for this endpoint should be one selected in the drop down menu.

**Comment by bkyryliuk** [comment]
@vera-liu  - please add a test plan to features
it simplifies the review process and explains how the feature was tested

**Comment by vera-liu** [comment]
Added @bkyryliuk

**Comment by vera-liu** [comment]
After discussing with Max, I add schema to temp_table_name before adding query, so that the actual schema used is shown in output column of Query History. PTAL @bkyryliuk

File: superset/sql_lab.py [hunk 2]
```diff
@@ -105,7 +103,7 @@ def handle_error(msg):
                 query.user_id,
                 start_dttm.strftime('%Y_%m_%d_%H_%M_%S'))
         executed_sql = create_table_as(
-            executed_sql, query.tmp_table_name, database.force_ctas_schema)
+            executed_sql, query.tmp_table_name)
```
Comment (bkyryliuk) [inline superset/sql_lab.py]: please add a comment that tmp_table_name includes schema here

File: superset/views.py [hunk 3]
```diff
@@ -2383,6 +2383,14 @@ def table_accessible(database, full_table_name, schema_name=None):
                 get_datasource_access_error_msg('{}'.format(rejected_tables)))
         session.commit()
 
+        select_as_cta = request.form.get('select_as_cta') == 'true'
+        tmp_table_name = request.form.get('tmp_table_name')
+        if select_as_cta and mydb.force_ctas_schema:
+            tmp_table_name = '{}.{}'.format(
+                mydb.force_ctas_schema,
+                tmp_table_name
```
Comment (bkyryliuk) [inline superset/views.py]: could u please add a unit test to the celery tests?

**Comment by bkyryliuk** [comment]
LGTM

**Pull request merged**